src/Mod/PartDesign/App/FeatureHole.cpp

```diff
@@ -94,46 +94,49 @@ const Hole::ThreadDescription Hole::threadDescription[][171] =
-        { "M2.2", 	2.2,       0.45 },
-        { "M2.5", 	2.5,       0.45 },
-        { "M3", 	3.0,       0.50 },
-        { "M3.5", 	3.5,       0.60 },
-        { "M4", 	4.0,       0.70 },
-        { "M4.5", 	4.5,       0.75 },
-        { "M5", 	5.0,       0.80 },
-        { "M6", 	6.0,       1.00 },
-        { "M7", 	7.0,       1.00 },
-        { "M8", 	8.0,       1.25 },
-        { "M9", 	9.0,       1.25 },
-        { "M10", 	10.0,      1.50 },
-        { "M11", 	11.0,      1.50 },
-        { "M12", 	12.0,      1.75 },
-        { "M14", 	14.0,      2.00 },
-        { "M16", 	16.0,      2.00 },
-        { "M18", 	18.0,      2.50 },
-        { "M20", 	20.0,      2.50 },
-        { "M22", 	22.0,      2.50 },
-        { "M24", 	24.0,      3.00 },
-        { "M27", 	27.0,      3.00 },
-        { "M30", 	30.0,      3.50 },
-        { "M36", 	36.0,      4.00 },
-        { "M39", 	39.0,      4.00 },
-        { "M42", 	42.0,      4.50 },
-        { "M45", 	45.0,      4.50 },
-        { "M48", 	48.0,      5.00 },
-        { "M52", 	52.0,      5.00 },
-        { "M56", 	56.0,      5.50 },
-        { "M60", 	60.0,      5.50 },
-        { "M64", 	64.0,      6.00 },
-        { "M68", 	68.0,      6.00 },
+        { "M1", 	1.0,    0.25,   0.75 },
+        { "M1.1", 	1.1,    0.25,   0.85 },
+        { "M1.2", 	1.2,    0.25,   0.95 },
+        { "M1.4", 	1.4,    0.30,   1.10 },
+        { "M1.6", 	1.6,    0.35,   1.25 },
+        { "M1.8", 	1.8,    0.35,   1.45 },
+        { "M2", 	2.0,    0.40,   1.60 },
+        { "M2.2", 	2.2,    0.45,   1.75 },
+        { "M2.5", 	2.5,    0.45,   2.05 },
+        { "M3", 	3.0,    0.50,   2.50 },
+        { "M3.5", 	3.5,    0.60,   2.90 },
+        { "M4", 	4.0,    0.70,   3.30 },
+        { "M4.5", 	4.5,    0.75,   3.70 },
+        { "M5", 	5.0,    0.80,   4.20 },
+        { "M6", 	6.0,    1.00,   5.00 },
+        { "M7", 	7.0,    1.00,   6.00 },
+        { "M8", 	8.0,    1.25,   6.80 },
+        { "M9", 	9.0,    1.25,   7.80 },
+        { "M10", 	10.0,   1.50,   8.50 },
+        { "M11", 	11.0,   1.50,   9.50 },
+        { "M12", 	12.0,   1.75,   10.20 },
+        { "M14", 	14.0,   2.00,   12.00 },
+        { "M16", 	16.0,   2.00,   14.00 },
+        { "M18", 	18.0,   2.50,   15.50 },
+        { "M20", 	20.0,   2.50,   17.50 },
+        { "M22", 	22.0,   2.50,   19.50 },
+        { "M24", 	24.0,   3.00,   21.00 },
+        { "M27", 	27.0,   3.00,   24.00 },
+        { "M30", 	30.0,   3.50,   26.50 },
+        { "M33", 	33.0,   3.50,   29.50 },
+        { "M36", 	36.0,   4.00,   32.00 },
+        { "M39", 	39.0,   4.00,   35.00 },
+        { "M42", 	42.0,   4.50,   37.50 },
+        { "M45", 	45.0,   4.50,   40.50 },
+        { "M48", 	48.0,   5.00,   43.00 },
+        { "M52", 	52.0,   5.00,   47.00 },
+        { "M56", 	56.0,   5.50,   50.50 },
+        { "M60", 	60.0,   5.50,   54.50 },
+        { "M64", 	64.0,   6.00,   58.00 },
+        { "M68", 	68.0,   6.00,   62.00 },
      },
     /* ISO metric fine */

```

Comment:
I'd add a comment that the fine threads are equation-driven, something like:
```suggestion
    /* ISO metric fine (no core hole entry, it's calculated exactly by diameter - pitch) */
```